src/number_theory/modular_forms/slash_actions.lean

```diff
@@ -104,20 +100,94 @@ begin
   ring,
 end
 
+lemma neg_slash (k : ℤ) (A : GL(2, ℝ)⁺) (f : ℍ → ℂ) : (-f) ∣[k] A = - (f ∣[k] A) :=
+begin
+  ext1,
+  simp [slash],
+end
```

Comment:
```suggestion
funext $ by simp [slash]
```
Might as well make this a term-mode proof, since it's trivial.  Same for `one_slash`.